### PR TITLE
Numpy add numpy op bitwise_xor

### PR DESCRIPTION
**Pull request by gyshi**
## Description ##
(Brief description on what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: src/operator/numpy/np_elemwise_binary_op.cc [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*
```
Comment (haojin2) [inline src/operator/numpy/np_elemwise_binary_op.cc]: Move all the code to `np_elemwise_broadcast_op*` files.

File: src/operator/numpy/np_elemwise_binary_op.cu [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#include "../operator_common.h"
+#include "../tensor/elemwise_binary_op.h"
+#include "../tensor/elemwise_binary_broadcast_op.h"
+
+namespace mxnet {
+namespace op {
+
+NNVM_REGISTER_OP(_np_bitwise_xor)
+.set_attr<FCompute>("FCompute<gpu>", BinaryBroadcastCompute<gpu,
+    mshadow_op::bitwise_xor>);
```
Comment (haojin2) [inline src/operator/numpy/np_elemwise_binary_op.cu]: Alignments
```c++
.set_attr<FCompute>("FCompute<gpu>",
                    BinaryBroadcastCompute<gpu, mshadow_op::bitwise_xor>);
```

File: src/operator/numpy/np_elemwise_binary_op.cc (hunk 1 above)
Comment (gyshi) [reply src/operator/numpy/np_elemwise_binary_op.cc]: resolved

**Comment by haojin2** [comment]
`bitwise_xor` has been added in #15840, closing this PR.

**Pull request closed without merging**